pom.xml

```diff
@@ -216,25 +216,14 @@
             <artifactId>jaxb-api</artifactId>
             <version>2.3.0</version>
         </dependency>
-        <!-- https://mvnrepository.com/artifact/javax.xml.bind/jaxb-api -->
-        <dependency>
 
-            <groupId>javax.xml.bind</groupId>
-            <artifactId>jaxb-api</artifactId>
-            <version>2.3.0-b170201.1204</version>
-        </dependency>
```

Comment:
I think those libraries are used when going through xmls from GBF...

Reply:
What's your concern here? The dependency wasn't removed- just the duplicate pre-release artifact. Prior to this there, there were elements for both `javax.xml.bind:jaxb-api:2.3.0-b170201.1204` and `javax.xml.bind:jaxb-api:2.3.0` and